### ISSUE TITLE
Outer circle shows a small gap at the top

### ISSUE THREAD

**Issue by nexcel99** [nikitahl/svg-circle-progress-generator#2]
Hi Nikitahl,

A nice tool, quite handy!

I had an issue with the outer circle, showing a small gap at the top whem zooming in. I've resolved it by removing the stroke-dasharray and stroke-dashoffset attributes. They seem to serve no purpose at the outer circle.

Cheers, Alex

**Comment by nikitahl** [comment]
Hello!
Thanks for noticing. I will check it.

**Comment by nikitahl** [comment]
Fixed with this [commit](https://github.com/nikitahl/svg-circle-progress-generator/commit/278796a476fc4a66c172b8b53ee27a55fe45c1ef).

The issue was replicated on a zoomed SVG (zoom in the browser) with a square shape and increased size and width.

I've removed the `stroke-dasharray` and the `stroke-dashoffset` attributes on the outer circle.
Also, I've added a 1 digit to a circumference property which is used in the calculation of a `stroke-dasharray` attribute of the inner circle (progress), to fix the same issue (a small gap on a zoomed SVG.)
